site/specs-source/voice.json

```diff
@@ -387,6 +387,99 @@
         }
       }
     },
+    "/api/v2/accounts/{accountId}/statistics": {
+      "get": {
+        "tags": [
+          "Calls"
```

Comment:
Do we want to use a new `Statistics` tag for these endpoints? this determines how theyre grouped in the documentation.

Reply:
I think it should be tagged with `Statistics`. @sspeth1 mentioned that the plan will be to have conference statistics and general voice statistics so it would be better to give it a separate category from the start.

Reply:
That makes sense. Done